### DISCUSSION TITLE
set up session timeout limit

### DISCUSSION THREAD

**Discussion by Aidoc-IT-team** [getredash/redash#6875] [Q&A]
Hi,

We have integrated Okta into the Redash system as an identity provider. However, we have noticed that there is no timeout limit session, which allows members to stay connected indefinitely without requiring them to log in again from Okta.

Could someone assist us in setting up a session timeout limit?
 
Thank you!

**Comment by zachliu** [comment]
don't we have a `REDASH_SESSION_EXPIRY_TIME` (default value 6 hours) and a `REDASH_REMEMBER_COOKIE_DURATION` (default 31 days)?
